haiku-apps/textworker/textworker-0.9.1.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A small utility for text transformations and cleanup"
+DESCRIPTION="TextWorker is a lightweight Haiku-native utility that performs \
+various transformations on text. It supports operations like case conversion, line sorting, trimming, prefix/suffix edits, and more.
+
+This is beta software provided for testing purposes and may contain bugs or incomplete features. Feedback is welcome."
+HOMEPAGE="https://github.com/dospuntos/TextWorker"
+COPYRIGHT="2025 Johan Wagenheim"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="49976a3f19e71920e3e045412112878786d6f6228e689770b2e9ba7d41c81541"
+SOURCE_DIR="TextWorker-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	textworker$secondaryArchSuffix = $portVersion
+	app:TextWorker = $portVersion
+	"
+REQUIRES="
+	haiku$secundaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	icu74${secondaryArchSuffix}_devel
```

Comment:
Use library references instead of package names:

```suggestion
	devel:libicui18n$secondaryArchSuffix >= 74
	devel:libicuuc$secondaryArchSuffix >= 74
```